packages/components/src/components/SquigglePlayground.tsx

```diff
@@ -142,80 +156,221 @@ let SquigglePlayground: FC<PlaygroundProps> = ({
       setImportsAreValid(false);
     }
   };
+
+  // className="text-gray-400 group-hover:text-gray-500 -ml-0.5 mr-2 h-4 w-4"
+  let tab = (key, iconName) => {
```

Comment:
In my experience it's (almost) always better to use real react components instead of anon functions:
- better debugging in browser
- better type checks (React.FC must return an Element or null)

Also, arguments are untyped.